From: Max Filippov <jcmvbkbc@gmail.com>
Subject: Don't overwrite CFLAGS when configured with --enable-debug

This drops architecture-specific ABI flags, which may be important.

Signded-off-by: Max Filippov <jcmvbkbc@gmail.com>

--- jasper-1.900.1/configure.ac.orig	2015-05-18 22:27:53.057512760 +0300
+++ jasper-1.900.1/configure.ac	2015-05-18 22:28:36.090415422 +0300
@@ -327,7 +327,7 @@
 	AC_DEFINE(DEBUG)
 	AC_DEFINE(DEBUG_OVERFLOW)
 	if test "$GCC" = yes; then
-		CFLAGS="-g -O0"
+		CFLAGS="$CFLAGS -g -O0"
 	fi
 	;;
 no)
@@ -357,7 +357,7 @@
 case "${enableval}" in
 yes)
 	if test "$GCC" = yes; then
-		CFLAGS="-g -O0"
+		CFLAGS="$CFLAGS -g -O0"
 	fi
 	;;
 no)
